src/components/data-management/ProjectMenu.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Row, Typography, Space, Button, Col,
+} from 'antd';
+import {
+  updateProject,
+} from '../../redux/actions/projects'; import DownloadData from './DownloadData';
+import fileUploadSpecifications from '../../utils/upload/fileUploadSpecifications';
+import UploadStatus from '../../utils/upload/UploadStatus';
+
+const { Title, Text, Paragraph } = Typography;
+
+const ProjectMenu = (props) => {
+  const {
+    activeProjectUuid, createMetadataColumn, isAddingMetadata,
+    setUploadModalVisible, openAnalysisModal,
```

Comment:
take a look if any of these functions aren't bettter to be moved to this file and not imported as props, dimilarly to what you did with `checkLaunchAnalysis`